src/pygambit/tests/test_behav.py

```diff
@@ -1,703 +1,701 @@
-            gbt.Rational("3/1")
-        )
-        assert (
-            self.profile_rational.action_value(self.game.players[1].infosets[0].actions[0]) ==
-            gbt.Rational("3/1")
-        )
-        assert (
-            self.profile_rational.action_value(self.game.players[1].infosets[0].actions[1]) ==
-            gbt.Rational("3/1")
-        )
-        assert (
-            self.profile_rational.action_value(self.game.players[2].infosets[0].actions[0]) ==
-            gbt.Rational("7/2")
-        )
-        assert (
-            self.profile_rational.action_value(self.game.players[2].infosets[0].actions[1]) ==
-            gbt.Rational("3/1")
-        )
-
-    def test_action_value_by_string(self):
-        """Test to retrieve expected payoff associated to an action
-        by string values
-        """
-        assert self.profile_double.action_value("U1") == 3.0
-        assert self.profile_double.action_value("D1") == 3.0
-        assert self.profile_double.action_value("U2") == 3.0
-        assert self.profile_double.action_value("D2") == 3.0
-        assert self.profile_double.action_value("U3") == 3.5
-        assert self.profile_double.action_value("D3") == 3.0
-
-        assert self.profile_rational.action_value("U1") == gbt.Rational("3/1")
-        assert self.profile_rational.action_value("D1") == gbt.Rational("3/1")
-        assert self.profile_rational.action_value("U2") == gbt.Rational("3/1")
-        assert self.profile_rational.action_value("D2") == gbt.Rational("3/1")
-        assert self.profile_rational.action_value("U3") == gbt.Rational("7/2")
-        assert self.profile_rational.action_value("D3") == gbt.Rational("3/1")
-
-    def test_regret(self):
-        for profile in [self.profile_double, self.profile_rational]:
-            for player in self.game.players:
-                for infoset in player.infosets:
-                    for action in infoset.actions:
-                        assert (
-                            profile.regret(action) ==
-                            max(profile.action_value(a) for a in infoset.actions) -
-                            profile.action_value(action)
-                        )
-
-    def test_node_value(self):
-        # Another good node_value test (to be written!) is its martingale property: it should
-        # be the expected value of its children's node_values, given the probability
-        # distribution at the node.
-        for profile in [self.profile_double, self.profile_rational]:
-            for player in self.game.players:
+# tolerance for floating point assertions
+TOL = 1e-13
+
+
+def _create_mixed_behav_game():
+    """
+    Returns
+    -------
+    Game
+        Three-player extensive form game: binary tree with 3 infomation sets, one per player,
+        with 1, 2, and 4 nodes respectively
+    """
+    return gbt.Game.read_game("test_games/mixed_behavior_game.efg")
+
+
+def _create_complicated_exten_game():
```

Comment:
Suggest writing out "extensive" (seems silly to have a long function already and then bother to save just a few letters)